src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -491,7 +491,7 @@ private function generateTokenForUser(UserInterface $user)
      */
     private function generateTokenHash(string $token): string
     {
-        return \hash('sha1', $this->secret . $token);
+        return \hash('sha1', $this->secret .'%'. $token);
```

Comment:
The generated hash still has to stay the same otherwise all password reset tokens would be invalidated.